spring-integration-file/src/main/java/org/springframework/integration/file/remote/RemoteFileUtils.java

```diff
@@ -76,4 +76,14 @@ private RemoteFileUtils() {}
 		}
 	}
 
+	/**
+	 * Invoke this method to clean up and close the session after using {@link Session#readRaw(String)}.
+	 * @param session The session.
+	 * @throws IOException if an IO error occurs.
+	 */
+	public static <F> void closeSession(Session<F> session) throws IOException {
+		session.finalizeRaw();
+		session.close();
```

Comment:
Iin the continuation of the SpEL discussion below how about to encapsulate the `finalizeRaw()` call to the `close()`?
Having that we can avoid this extra code and there won't be need to teach end-user for that complex SpEL :smile: .